applications/callflow/src/cf_event_handler_sup.erl

```diff
@@ -44,6 +44,13 @@ new(Name, M, A) ->
 workers() ->
     [Pid || {_, Pid, 'worker', [_]} <- supervisor:which_children(?MODULE)].
 
+-spec worker(ne_binary()) -> api_pid().
+worker(Name) ->
+    case [Pid || {Worker, Pid, 'worker', [_]} <- supervisor:which_children(?MODULE), Worker =:= Name] of
```

Comment:
why would you use that ? `workers()`returns list of pids, you'll never find `Name` in it.